docs/release-notes/log-signal.rst

```diff
@@ -0,0 +1,39 @@
+:orphan:
+
+**New Features**
+
+-  Experiments: ``log_policies`` now have a ``name`` field. When a log policy matches, its name will
+   be displayed as a label in the WebUI, allowing for easy identification of specific issues during
+   a run. These labels will appear in both the run table and run detail views.
+
+   It has a new format. ``name`` is required. ``pattern`` and ``action`` are optional. To make
+   things simpler, user no longer needs to specify the ``type`` field to set an action. For example:
```

Comment:
> It has a new format.  `name` is required, and `action` should be a plain string.  See <link to docs> for more details.

Then skip the rest of the text.

The release note should be a brief note and interested users should go to the main docs.